beta/src/content/apis/react/createFactory.md

```diff
@@ -4,7 +4,7 @@ title: createFactory
 
 <Deprecated>
 
-This API will be removed in a future major version of React. Use [JSX](/learn/writing-markup-with-jsx) or [`createElement`](/api/react/createElement) instead.
+This API will be removed in a future major version of React. [See the alternatives.](#alternatives)
```

Comment:
```suggestion
This API will be removed in a future major version of React. [See the alternatives](#alternatives).
```
nit 😄

Reply:
nah, i prefer to keep punctuation inside the links